cogs/guess.py

```diff
@@ -64,20 +64,20 @@ def __init__(self, bot):
             if failed == 0:        
                 await self.bot.reply("You won! Good job! :aquaThumbsUp:")  
             """ exit the script"""
-            break              
-
-            print
+            
+            break
+            
             """ ask the user go guess a character"""
+            await self.bot.say("lower case letters please~~!")
             guess = self.bot.discord.wait_for_message(timeout=None, author=None, channel=None, content=None, check=None)
             """ set the players guess to guesses"""
             guesses += guess
             """ incorrect letter"""
-            await self.bot.say("lower case letters please~~!")
             if guess not in secret:
                 turns -= 1        
                 await self.bot.say("Wrong")    
             """ number of turns left"""
-            await self.bot.say("You have", + turns, "more guesses") 
+            await self.bot.say("You have", +turns, "more guesses") 
```

Comment:
I'm kind of wondering what that plus sign does. Is it necessary?

Reply:
They're doing string concatenation weirdly lol